editor.planx.uk/src/@planx/components/MapAndLabel/test/public.test.tsx

```diff
@@ -386,8 +386,10 @@ describe("copy feature select", () => {
     await user.click(copyInput);
 
     const listItemTwo = getByRole("option", { name: "Tree 2" });
+    const copyButton = getByTestId("copyButton");
```

Comment:
Spent a while trying to grab this via `getByRole("button", { name: "Copy" });` but RTL wasn't finding it.